### ISSUE TITLE
Can you stop it from limiting how many texts for a number I can send a day

### ISSUE THREAD

**Issue by pycoder74** [typpo/textbelt#171]

**Comment by ryanmr82** [comment]
i keep trying to contact texbelt support for 2 weeks now and i never get a reply from them. i keep getting daily limit errors especially while im doing testing for my application. anyone had any luck with getting a response from their support email? they never respond  to me, kind of frustrating.

**Comment by ryanmr82** [comment]
i just filed a paypal dispute and i'm changing to another provider. textbelt customer support is non existent and they are riddled with issues like random cell provider blocking and daily limit issues when i'm not even sending more than 30 texts a day while i do user acceptance testing.

**Comment by daluu** [comment]
So just to be clear, this is with respect to the public paid hosted service? Not talking about the free self hosted version on github here that you can host yourself? As the latter has no limits I believe or that you can adjust the limits yourself in the config/code.

For the former, not sure what channels/outlets you can go for support, the github issues here are for the free self hosted version regarding issues/bugs with the code that you run.

**Comment by ryanmr82** [comment]
Yes, correct. I'm referring to the paid service where you can buy bundles, e.g. 700 SMS messages for $10. I wasn't aware that there is an option to have it run locally on your server, I may look into this further, but I'm not sure how that would work or if anybody has successfully done it. My original inquiry and search online was to figure out if anybody had any luck contacting their support service.

But looking at the subject of this Github thread here, it says "Can you stop it from limiting how many texts for a number I can send a day?" So if this individual, Ghost, is referring to the local hosted version, it seems like he's saying also that there is a limit on how many can be sent per day. I could be wrong, maybe I'm not sure what Ghost is talking about. Or is Ghost talking about the paid version, which is what I originally thought because I'm currently paying for this service. That's why I originally left the comment here. Let me know if I can add more clarity.

**Comment by ryanmr82** [comment]
Update, filing a dispute got their intention. Support responded to me after that. They said that my email was flagged as "ChatGPT spam" not sure what that means? I let them know if they are able to address the daily limit that I would continue to pay for them. So far no reply. I'll update more once I hear back (that is if I hear back) to see if they are able to provide good support.

**Comment by BailinSong** [comment]
I encountered the same problem, have you changed to another supplier?

**Comment by thaguitarist** [comment]
Hello
I know this github is for the free version but just letting you know that we also have the same issue. 
We have integrated their API into the software we provide to our customers so they can use Textbelt to send SMS to their own customers and in the last few months, a lot of our customers have been facing daily limit issues (which goes against their faq stating there is no daily limit), with no reply from Textbelt.

My tech did tell me he used chatgpt for some translation and syntax for the email he wrote them so there might be a possibility in our case that their ChatGPT spam thing made them not receiving our numerous emails..

**Comment by daluu** [comment]
@typpo is there any appropriate avenue for users to contact/post regarding the public paid service issue described here? This isn't the right place and gets no traction here anyways.

**Comment by daluu** [comment]
> ..., with no reply from Textbelt

This makes me think of the demise of AutoSMS, a free SMS service similar to TextBelt but which required CAPTCHA handling to use. It just closed shop one day without much advanced warning.

TextBelt service used to be free with limits, then became paid service for those not self hosting. Might it shut down one day too?
